src/elements/e-table.vue

```diff
@@ -117,22 +328,85 @@
          * @type {Boolean} Holds to sort direction in case a 'sortBy' is active.
          */
         sortAscending: true,
+
+        /**
+         * @type {Boolean} Holds a flag if row links should be enabled.
+         */
+        enableRowLinks: false,
+
+        /**
+         * @type {Boolean} Holds a flag if there is currently a text selection inside the component.
+         */
+        hasSelection: false,
+
+        /**
+         * @type {Number} Holds the height of the sorting list toggle button.
+         */
+        toggleButtonHeight: 0,
+
+        /**
+         * Row items that should be displayed in expanded state.
+         */
+        expandedRows: [],
       };
     },
 
     computed: {
+      /**
+       * Checks if current viewport is mobile (<= sm).
+       *
+       * @returns {Boolean}
```

Comment:
Please remove and use TS typing (for parameters and returns) instead. (For all functions)